cmd/tetragon/main.go

```diff
@@ -368,6 +369,8 @@ func execute() error {
 			}
 			log.WithField(keyConfigDir, configDir).Info("Loaded config from directory")
 		}
+		replacer := strings.NewReplacer("-", "_")
+		viper.SetEnvKeyReplacer(replacer)
```

Comment:
I thought viper did the "-" -> "_" translation on its own, is this not the case?

Reply:
@kkourt not by default according to their docs, but they have this in one of their examples.

Reply:
Near the end of this section https://github.com/spf13/viper#working-with-environment-variables

Reply:
@kkourt yep seems not the case! it doesn't handle parameters with "-"... by default but with that replacer it works, also I checked cilium and they do this manually, replace all "-" with "_" concat then env prefix then uppercase...

Reply:
@kkourt found this: https://github.com/spf13/viper/blob/master/viper_test.go#L720